tests/scripts/thread-cert/message.py

```diff
@@ -189,7 +189,10 @@ def assertMleMessageContainsOptionalTlv(self, tlv_class_type):
                 contains_tlv = True
                 break
 
-        print("MleMessage doesn't contain optional TLV: {}".format(tlv_class_type))
+        if contains_tlv == True:
```

Comment:
sorry. I don't think "if contains_tlv == True" has a problem. And it is good for code readability?